### DISCUSSION TITLE
Played waveform width is not matching my calculation can you help with this ?

### DISCUSSION THREAD

**Discussion by vishal-4way** [katspaugh/wavesurfer.js#2674] [Q&A]
![Screenshot_2](https://user-images.githubusercontent.com/105101029/218103989-a18e9e37-40d1-4315-b68f-a8197df9e458.png)

minPxPerSec : 1400
waveform.getDuration() > 197.53902083333332
total width of waveform > 276555px correct
waveform.getCurrentTime() > 31.07933333333327
width of color waveform > 43511px 

my PxPerSec : 1400
my current time is 30.950 

because of this my playhead position is not matching with the waveform cursor .

Is there any way so that I can limit the waveform calculation upto 10ms only .

**Comment by entonbiba** [comment]
@vishal-4way do you have a link so we can see and what version you using?
